examples/navier-stokes/densitycurrent.h

```diff
@@ -102,7 +103,9 @@ static int Exact_DC(CeedInt dim, CeedScalar time, const CeedScalar X[], CeedInt
   // -- Potential temperature, density current
   CeedScalar rr[3] = {x - center[0], y - center[1], z - center[2]};
   // (I - q q^T) r: distance from dc_axis (or from center if dc_axis is the zero vector)
-  for (int i=0; i<3; i++) rr[i] -= dc_axis[i] * (dc_axis[0]*rr[0] + dc_axis[1]*rr[1] + dc_axis[2]*rr[2]);
+  for (int i=0; i<3;
+       i++) rr[i] -= dc_axis[i] * (dc_axis[0]*rr[0] + dc_axis[1]*rr[1] +
```

Comment:
probably this should have not been accepted this way, with the for loop statement broken on two lines